includes/class-activitypub.php

```diff
@@ -738,4 +740,16 @@ public static function updated_postmeta( $meta_id, $object_id, $meta_key, $meta_
 			\delete_post_meta( $object_id, 'activitypub_content_visibility' );
 		}
 	}
+
+	/**
+	 * Register some Mastodon oEmbed providers.
+	 */
+	public static function register_oembed_providers() {
+		\wp_oembed_add_provider( '#https?://mastodon\.social/(@.+)/([0-9]+)#i', 'https://mastodon.social/api/oembed', true );
+		\wp_oembed_add_provider( '#https?://mastodon\.online/(@.+)/([0-9]+)#i', 'https://mastodon.online/api/oembed', true );
+		\wp_oembed_add_provider( '#https?://mastodon\.cloud/(@.+)/([0-9]+)#i', 'https://mastodon.cloud/api/oembed', true );
+		\wp_oembed_add_provider( '#https?://mstdn\.social/(@.+)/([0-9]+)#i', 'https://mstdn.social/api/oembed', true );
+		\wp_oembed_add_provider( '#https?://mastodon\.world/(@.+)/([0-9]+)#i', 'https://mastodon.world/api/oembed', true );
+		\wp_oembed_add_provider( '#https?://mas\.to/(@.+)/([0-9]+)#i', 'https://mas.to/api/oembed', true );
+	}
```

Comment:
Would it be possible to just add these dynamically based on what posts the user tries to embed? Assume that they use this oembed pattern? 

Otherwise anybody trying to embed a post that's not in the list won't work.

Reply:
The code falls back to Activity-JSON parsing if the embed does not return a result.

Reply:
Oh! The Activity JSON fallback isn't working in my local environment so I didn't know it was there. Will test in a different env 👍🏻